docs/README.md

```diff
@@ -0,0 +1,275 @@
+| Voyager1          | N/A                   | Croatian, Italian                             |
+| vesgo             | N/A                   | Portugese                                     |
+| Varviar           | N/A                   | N/A                                           |
+| V543000           | Vaclav Benc           | N/A                                           |
+| uzurpator         | N/A                   | N/A                                           |
+| UseYourIllusion   | N/A                   | Indonesian                                    |
+| Trond             | N/A                   | Norwegian (bokmal)                            |
+| Tenebrae          | N/A                   | Slovak                                        |
+| Telk              | N/A                   | Korean                                        |
+| telanus           | N/A                   | Afrikaans                                     |
+| TadeuszD          | N/A                   | Polish                                        |
+| Supercheese       | N/A                   | English (US) & Latin                          |
+| stravagante       | N/A                   | Serbian                                       |
+| Stabilitronas     | N/A                   | Lithuanian                                    |
+| Spaz O Mataz      | N/A                   | N/A                                           |
+| Soeb              | Stanislaw Gackowski   | N/A                                           |
+| Snail             | Jacopo Coletto        | N/A                                           |
+| skidd13           | Benedikt Brüggemeier  | N/A                                           |
+| siu238X           | N/A                   | Chinese (simplified), Chinese (traditional)   |
+| SilverSurveferZzZ | N/A                   | Spanish                                       |
+| Rubidium          | Remko Bijker          | N/A                                           |
+| Roujin            | Manuel Wolf           | N/A                                           |
+| Red*Star          | David Krebs           | N/A                                           |
+| Raumkraut         | Mel Collins           | N/A                                           |
+| Purno             | Mark Leppen           | N/A                                           |
+| planetmaker       | Ingo van Borstel      | German                                        |
+| PikkaBird         | David Dallaston       | N/A                                           |
+| Phreeze           | N/A                   | Luxemburgean                                  |
+| Pekkape01         | N/A                   | Swedish                                       |
+| PaulC             | Paul Charlesworth     | N/A                                           |
+| orduge            | Owen Ridge            | N/A                                           |
```

Comment:
Owen's handle and surname are both misspelled here, they're correct in the original file

Reply:
The new commit https://github.com/OpenTTD/OpenGFX/pull/2/commits/449cad069b3c403059ab242f9d5c54f114039768 has fixed his handle and surname. Thanks for pointing this out!